scprep/run/dyngen.py

```diff
@@ -0,0 +1,311 @@
+            cat('\n\tNumber of Targets:', num_targets)
+            cat('\n\tNumber of HKs:', num_hks, '\n')
+        }
+
+        init <- initialise_model(
+          backbone=backbone,
+          num_cells=num_cells,
+          num_tfs=num_tfs,
+          num_targets=num_targets,
+          num_hks=num_hks,
+          simulation_params=simulation_default(
+            census_interval=as.double(simulation_census_interval),
+            kinetics_noise_function = kinetics_noise_simple(mean=1, sd=0.005),
+            ssa_algorithm = ssa_etl(tau=300/3600),
+            compute_cellwise_grn=compute_cellwise_grn,
+            compute_rna_velocity=compute_rna_velocity),
+          num_cores = n_jobs,
+          download_cache_dir=NULL,
+          verbose=verbose
+        )
+        out <- generate_dataset(init)
+        data <- list(cell_info = as.data.frame(out$dataset$cell_info),
+             expression = as.data.frame(as.matrix(out$dataset$expression)))
+
+        if (compute_cellwise_grn) {
+            data[['bulk_grn']] <- as.data.frame(out$dataset$regulatory_network)
+            data[['cellwise_grn']] <- as.data.frame(out$dataset$regulatory_network_sc)
+        }
+        if (compute_rna_velocity) {
+            data[['rna_velocity']] <- as.data.frame(as.matrix(out$dataset$rna_velocity))
+        }
+
+        data
+    """,
+)
+
+
+def install(
+    lib=None,
+    dependencies=None,
+    update=False,
+    repos="http://cran.us.r-project.org",
+    build_vignettes=False,
+    force=False,
+    verbose=True,
+):
+    """Install Dyngen Github repository.
+
+    Parameters
+    ----------
+    lib: string
+        Directory to install the package.
+        If missing, defaults to the first element of .libPaths().
+    dependencies: boolean, optional (default: None/NA)
+        When True, installs all packages specified under "Depends", "Imports",
+        "LinkingTo" and "Suggests".
+        When False, installs no dependencies.
+        When None/NA, installs all packages specified under "Depends", "Imports"
+        and "LinkingTo".
+    update: string or boolean, optional (default: False)
+        One of "default", "ask", "always", or "never". "default"
+        Respects R_REMOTES_UPGRADE environment variable if set, falls back to "ask" if unset.
+        "ask" prompts the user for which out of date packages to upgrade.
+        For non-interactive sessions "ask" is equivalent to "always".
+        TRUE and FALSE are also accepted and correspond to "always" and "never" respectively.
+    repos: string, optional (default: "http://cran.us.r-project.org"):
+        R package repository.
+    build_vignettes: boolean, optional (default: False)
+        Builds Github vignettes.
+    force: boolean, optional (default: False)
+        Forces installation even if remote state has not changed since previous install.
+    verbose: boolean, optional (default: True)
+        Install script verbosity.
+    """
+
+    r_function.install_github(
```

Comment:
Per failing tests [here](https://github.com/aarthivenkat/scprep/actions/runs/579707090) it looks like we have to also install `dynwrap` manually. Posted an issue: https://github.com/dynverse/dyngen/issues/28